website/src/documentation/transforms/java/aggregation/mean.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: section
+title: "Mean"
+permalink: /documentation/transforms/java/aggregation/mean/
+section_menu: section-menu/documentation.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Mean
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/transforms/Mean.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br>
+Transforms for computing the arithmetic mean of the elements in a collection,
+or the mean of the values associated with each key in a collection of key-value pairs.
+
+* `Mean.globally()` returns a transform that then returns a collection whose contents is the mean of the input collection's elements. If there are no elements in the input collection, it returns 0.
+* `Mean.of()` is a `Combine.CombineFn` that computes the arithmetic mean of the elements in a collection.
```

Comment:
Users don't have to use this `combineFn` directly. So I think this can be removed from here.

Reply:
Do you mean remove "is a `Combine.CombineFn` that" ?

Reply:
No. I mean remove this sentence about `Mean.of()` entirely. Because it is not really a `PTransform`.